tools/stress-cluster/chaos/README.md

```diff
@@ -76,9 +76,9 @@ kubectl create namespace <your alias>
 You will then need to build and push your container image to a registry the cluster has access to:
 
 ```
-az acr login -n stresstestregistry
-docker build . -t stresstestregistry.azurecr.io/<your name>/<test job image name>:<version>
-docker push stresstestregistry.azurecr.io/<your name>/<test job image name>:<version>
+az acr login -n stresstesttbiruti6oi24k
+docker build . -t stresstesttbiruti6oi24k.azurecr.io/<your name>/<test job image name>:<version>
+docker push stresstesttbiruti6oi24k.azurecr.io/<your name>/<test job image name>:<version>
```

Comment:
Not necessary, but if we want this to be dynamic so it doesn't break should we re-build the cluster:

```suggestion
# get registry in powershell
$registry = az acr list -g rg-stress-cluster-test --subscription "Azure SDK Developer Playground" -o json
$registryName = ($registry | ConvertFrom-Json).name

# get registry in bash
registry=$(az acr list -g rg-stress-cluster-test --subscription "Azure SDK Developer Playground" -o json)
registryName=$(echo $registry | jq -r '.[].name')

# Build and push development image to stress test cluster registry
docker build . -t $registryName.azurecr.io/<your name>/<test job image name>:<version>
docker push $registryName.azurecr.io/<your name>/<test job image name>:<version>